integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -928,6 +928,99 @@ public void testDomainServerPodRestart(String oldPropertyString, String newPrope
     logger.info("Done - testDomainServerPodRestart");
   }
 
+  /**
+   * Get runtime server yaml file and verify the changed property is in that file.
+   *
+   * @param changedProperty - the changed/added property
+   * @param serverName - server name that is being tested
+   * @throws Exception - test FAILURE Exception if the changed property is not found in the server
+   *     yaml file
+   */
+  public void findServerPropertyChange(String changedProperty, String serverName) throws Exception {
+    logger.info("Inside findServerPropertyChange");
+    // get runtime server pod yaml file
+    String outDir = BaseTest.getUserProjectsDir() + "/weblogic-domains/" + domainUid + "/";
+    StringBuffer command = new StringBuffer();
```

Comment:
As you are already executing a command, you can change it to
```
kubectl get pod <podname> -o yaml -n ns | grep changedProperty 
```
instead of saving the file and looking for a string later..

Reply:
Done